core/src/com/softwarearchitecture/game_server/buttons/Button.java

```diff
@@ -0,0 +1,80 @@
+package com.softwarearchitecture.game_server.buttons;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Rectangle;
+import com.badlogic.gdx.math.Vector3;
+
+public class Button {
```

Comment:
Denne burde kanskje implementere AbstractButton?

Reply:
I think the AbstractButton class is no longer in existance

Reply:
AbstractButton does not exist